test/account_test.rb

```diff
@@ -79,10 +79,10 @@ def app
       assert_match %r{/login$}, last_response.headers["Location"]
     end
 
-# proving VERY difficult to test in isolation, need a cookie
-=begin
     it "redirects to dashboard on a successful confirmation" do
-      stub_heroku_api
+      any_instance_of(Identity::Cookie) do |cookie|
+        stub(cookie).access_token { "abc123" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.